lib/index.js

```diff
@@ -15,10 +15,31 @@ $(document).ready(function() {
   fetchDiaryInfo();
   $('#previous-day').on('click', previousDay)
   $('#next-day').on('click', nextDay)
+  $('.food-options').on('click', '.delete', deleteMealFood)
   $('.add-food-to-meal').on('click', submitFoodToMeal)
   $('#diary-food-search').on('keyup', searchFoods);
 });
 
+function deleteMealFood () {
```

Comment:
Nice. This will make it way easier if we decide to add in some classes and rearrange some of the code.